packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1198,7 +1247,6 @@ describe("Runtime", () => {
 
 				return patched;
 			}
-
```

Comment:
SUPER nit: add this line back (only if you have to come back and make other changes). Just to avoid unnecessary churn in the diff

Reply:
Related: Consider moving the two "new" tests above down where the similar tests were below to minimize the diff.